bears/jinja2/Jinja2Bear.py

```diff
@@ -0,0 +1,335 @@
+import re
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result, RESULT_SEVERITY
+from coalib.results.Diff import Diff
+
+
+def generate_spacing_diff(file, filename, line, line_number,
+                          match_object, required_spacing):
+    '''
```

Comment:
we prefer `"""`, we did use `'''` as a "marking for translation" back in the old times when we attempted to make all those docs translatable. Nowadays we only use `'''` for legacy code and when we use `"""` inside the doc comment.